sw/device/tests/dif/dif_entropy_smoketest.c

```diff
@@ -58,8 +58,8 @@ bool test_main() {
     // wait for entropy to become available
     while (dif_entropy_read(&entropy, &entropy_data[i]) != kDifEntropyOk)
       ;
-    // LOG_INFO("received %x, expectecd %x", entropy_data[i],
-    // kExpectedEntropyData[i]);
+    //LOG_INFO("received %x, expectecd %x", entropy_data[i],
+    //         kExpectedEntropyData[i]);
```

Comment:
Remove if unable to use. Does this cause issues with dv?

Reply:
o no it doesn't, but i kind of assumed it could cause issues with verilator and others if there's a max time for CI test and I'm doing a bunch of prints.

Reply:
i'm just going to print all of it and see if CI complains.